lib/mayaUsd/fileio/shading/shadingModeUseRegistry.cpp

```diff
@@ -433,48 +433,81 @@ class UseRegistryShadingModeImporter {
             return MObject();
         }
 
+        MPlug surfaceOutputPlug, volumeOutputPlug, displacementOutputPlug;
```

Comment:
We now read all known inputs of the material, which is symmetric with what the exporter writes.